### DISCUSSION TITLE
Can I define default buttons in the toolbar for the Bard fieldtype?

### DISCUSSION THREAD

**Discussion by geertjanknapen1** [statamic/cms#7362] [Q&A] [answered]
Currently, when adding a Bard field to a blueprint, I have to manually select what buttons I want enabled in the toolbar (H1, H2, subscript, horizontal rule, etc.).

In my use-case, the buttons will almost always be the same, is there a way that those buttons can be pre-selected when I add a new Bard field to a blueprint, but still have the option to disable some buttons should that be needed?

Explained in an image, when I choose to add the Bard field, it comes with these buttons in the toolbar:
<img width="787" alt="image" src="https://user-images.githubusercontent.com/33750300/212862232-b45235da-8ab7-4ec8-a536-96cf7503abd2.png">

Am I able to define somewhere that it should come pre-selected with some buttons, like this:
<img width="799" alt="image" src="https://user-images.githubusercontent.com/33750300/212862434-dda74c3d-9604-46ea-b77b-70ab76db2d1d.png">

**Comment by jasonvarga** [comment, marked as answer]
Not at the moment. (But you can use a fieldset approach described by Jack below.)

**Reply by geertjanknapen1** [reply]
Thanks for the quick reply, is it planned? Because I can imagine that I am not the only one that would love this, or can I somehow change this behavior myself?

**Reply by jackmcdade** [reply]
It is not planned. We have no UI/UX patterns for "setting a default and then optionally overriding it" – it's a pretty complex thing to accomplish in an intuitive manner. The best thing you can do is to have a "default" Bard field in a fieldset so you can import it anywhere you want, and if you don't like those settings, create a new Bard field instead with whatever you need.

**Reply by geertjanknapen1** [reply]
Hey Jack, thanks for the reply!

I didn't think of the fieldset with a pre-defined Bard field yet, pretty good solution, thank you!
